changelogs/265-get_updated_droplet_info_on_create_wait_by_id.yaml

```diff
@@ -0,0 +1,5 @@
+bugfix:
+  - digital_ocean_droplet - fix regression where droplet info (eg networking)
```

Comment:
Please change "eg" into "for example," and move the period after `true` to the very end of the line (after the final parenthesis).